src/common/utxobased/info/bitcointestnet4.ts

```diff
@@ -0,0 +1,88 @@
+import { EdgeCurrencyInfo } from 'edge-core-js/types'
+
+import { CoinInfo, EngineInfo, PluginInfo } from '../../plugin/types'
+import {
+  legacyMemoInfo,
+  utxoCustomFeeTemplate,
+  utxoMemoOptions
+} from './commonInfo'
+
+export const currencyInfo: EdgeCurrencyInfo = {
+  assetDisplayName: 'Bitcoin Testnet4',
+  chainDisplayName: 'Bitcoin Testnet4',
+  canReplaceByFee: true,
+  currencyCode: 'TESTBTC',
+  customFeeTemplate: utxoCustomFeeTemplate,
+  memoOptions: utxoMemoOptions,
+  pluginId: 'bitcointestnet4',
+  walletType: 'wallet:bitcointestnet4',
+
+  // Explorers:
+  blockExplorer: 'https://mempool.space/testnet4/block/%s',
+  addressExplorer: 'https://mempool.space/testnet4/address/%s',
+  transactionExplorer: 'https://mempool.space/testnet4/tx/%s',
+
+  denominations: [
+    { name: 'TESTBTC', multiplier: '100000000', symbol: '₿' },
+    { name: 'mTESTBTC', multiplier: '100000', symbol: 'm₿' },
+    { name: 'bits', multiplier: '100', symbol: 'ƀ' },
+    { name: 'sats', multiplier: '1', symbol: 's' }
+  ],
+
+  // Deprecated:
+  ...legacyMemoInfo,
+  defaultSettings: {
+    customFeeSettings: ['satPerByte'],
+    blockbookServers: [
+      'wss://blockbook.tbtc-1.zelcore.io',
+      'wss://testnet4-explorer.wakiyamap.dev'
+    ],
+    enableCustomServers: false
+  },
+  displayName: 'Bitcoin Testnet4',
+  metaTokens: []
+}
+
+export const engineInfo: EngineInfo = {
+  formats: ['bip49', 'bip84', 'bip44', 'bip32'],
+  forks: ['bitcoincash', 'bitcoingold'],
```

Comment:
I forget why we have these forks defined in `bitcointestnet.ts`. Was this a mistake all along?

Reply:
Probably, but if it matches the other testnet it's probably fine